crates/blockchain/payload.rs

```diff
@@ -180,10 +180,14 @@ impl<'a> PayloadBuildContext<'a> {
         self.payload.header.number
     }
 
-    fn store(&self) -> &Store {
+    fn store(&self) -> Option<&Store> {
```

Comment:
would it make sense to return a Result instead of an Option so you can just use `?` from outside?

Reply:
is there really a case where this is None? It seems we end up adding a lot of defensive code for something that never happens